presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5662,4 +5662,19 @@ public void testMapUnionSumOverflow()
                 "select y, map_union_sum(x) from (select 1 y, map(array['x', 'z', 'y'], cast(array[null,30, 32760] as array<smallint>)) x " +
                 "union all select 1 y, map(array['x', 'y'], cast(array[1,100] as array<smallint>))x) group by y", ".*Value 32860 exceeds MAX_SHORT.*");
     }
+
+    @Test
+    public void testSingleItemCSE()
```

Comment:
You probably want to just name this as `testLargeRow`?

Reply:
Done